saos-webapp/src/main/webapp/WEB-INF/view/home/home.jsp

```diff
@@ -20,6 +21,7 @@
 		
 	</div>
 </div>
+ --%>
```

Comment:
I have commented this part, because maybe we would like to use it in the future. But I guess I can take it  from git repo.